similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -111,13 +115,19 @@ public void initialize() throws InitializationException
     @Override
     public List<PatientSimilarityView> findSimilarPatients(Patient referencePatient)
     {
-        return find(referencePatient, false);
+        return findSimilarPatients(referencePatient, null);
```

Comment:
why not simply `return find(referencePatient, null, false);`

Reply:
more modular, this method is a shortcut for `findSimilarPatients` for the case when we have no consents, so w edo exactly that here. The fact that that method in turn calls something else is its internal implementation which we dont care about here, maybe in the future that will change, but this way we will only have to change it in one place (ther,e not both there and here)